v0.5/audit/nvidia/TEST03/README

```diff
@@ -25,7 +25,8 @@ Part II
 Run the benchmark in the same manner as the original submission, once in AccuracyOnly mode and once in
 SubmissionOnly mode. Ensure that accuracy.txt is generated along with the other mlperf_log_* logs.  
```

Comment:
@georgelyuan It's not clear to me why running in the `SubmissionOnly` mode is required, especially since the official recommendation was **not** to use it for v0.5?